lib/versionbot.ts

```diff
@@ -85,20 +85,20 @@ export class VersionBot extends GithubBot.GithubBot {
             {
                 events: [ 'pull_request' ],
                 name: 'CheckVersionistCommitStatus',
-                suppressionLabels: [ 'flow/no-version-checks' ],
+                suppressionLabels: [ 'procbots/versionbot/no-checks' ],
                 workerMethod: this.checkVersioning
             },
             {
                 events: [ 'pull_request' ],
                 name: 'CheckVersionistCommitStatus',
-                suppressionLabels: [ 'flow/no-version-checks' ],
-                triggerLabels: [ 'version/check-commits' ],
+                suppressionLabels: [ 'procbots/versionbot/no-checks' ],
+                triggerLabels: [ 'procbots/versionbot/check-commits' ],
                 workerMethod: this.checkVersioning
```

Comment:
Yeah, this is a hang-up from when I changed around where labels/events were filtered. Good catch, the second is now completely obsolete.